docs/customization/customizing-your-storefront.mdx

```diff
@@ -0,0 +1,170 @@
+application. If you're familiar with customizing views and assets on a regular Rails app, you
+already have the knowledge to customize the frontend of a Solidus application.
+
+For the rest of the guide, we'll assume that you have a running Solidus application with
+SolidusStarterFrontend as your storefront.
+
+## Editing a view
+
+If we inspect the home page, we'll see that the hero container has the following HTML code:
+
+```html
+<div class="hero__container">
+  <div class="hero__message">
+    <h1 class="hero-title">
+      The only eCommerce platform you’ll ever need.
+    </h1>
+
+    <p class="subtitle">
+      Build, customize and scale your store with no limits or license fees.
+      Solidus is the free, open-source eCommerce framework for
+      digitally-native brands, fast-growing online businesses and pragmatic
+      developers.
+    </p>
+  </div>
+</div>
+```
+
+If we search for views with the CSS class `"hero__container"` in your Solidus app, we find
+`app/views/spree/home/_hero.html.erb`. This partial was copied to your app during the installation
+of SolidusStarterFrontend.
+
+Let's use the [`image_tag`][image_tag] helper to try and add a placeholder for the hero image in
+that partial:
+
+```html
+<div class="hero__container">
+  <%= image_tag('hero-image') %>
+  <div class="hero__message">
+    ...
+  </div>
+</div>
+```
+
+If we test it out, we'll find that we need to add the hero image to the asset pipeline:
+
+![Missing hero image error](/img/customizing-your-storefront/missing-hero-image-error.png)
+
+## Running the frontend specs
+
+If you try and run the [RSpec][] test suite at this point, you may also see some specs fail because
+of this error:
+
+```
+$ bundle exec rspec spec/requests/spree/home_spec.rb
+
+...
+
+Failures:
+
+  1) Home layout provides current user to the searcher class
+     Failure/Error: <%= image_tag('hero-image') %>
+
+     ActionView::Template::Error:
+       The asset "hero-image" is not present in the asset pipeline.
+     # ./app/views/spree/home/_hero.html.erb:3:in `_app_views_spree_home__hero_html_erb___8794442347239906_218740'
+     # ./app/views/spree/home/index.html.erb:1:in `_app_views_spree_home_index_html_erb__1597336803422850061_218720'
+     # ./spec/requests/spree/home_spec.rb:18:in `block (2 levels) in <top (required)>'
+     # ------------------
+     # --- Caused by: ---
+     # Sprockets::Rails::Helper::AssetNotFound:
+     #   The asset "hero-image" is not present in the asset pipeline.
+     #   ./app/views/spree/home/_hero.html.erb:3:in `_app_views_spree_home__hero_html_erb___8794442347239906_218740'
+
+...
+
+```
+
+When you installed SolidusStarterFrontend to your app, you also received some some specs for testing
+the frontend. These specs are part of the SolidusStarterFrontend test suite itself, that is, they
+are used in testing the SolidusStarterFrontend project. By copying these specs to your app, you get
+a good starting test suite for testing your application.
```

Comment:
```suggestion
a good starting point for testing your application.
```

A lot of "test" or variations in the sentence. Maybe we can get rid of this one :slightly_smiling_face: